packages/web/src/components/comments/CommentThread.tsx

```diff
@@ -97,37 +97,45 @@ export const CommentThread = ({ commentId }: { commentId: ID }) => {
                 : messages.hideReplies}
             </PlainButton>
           </Box>
-          {hiddenReplies[rootComment.id] ? null : (
-            <Flex
-              direction='column'
-              gap='l'
-              as='ul'
-              aria-label={messages.replies}
-            >
-              {replies.map((reply: ReplyComment) => (
-                <Flex w='100%' key={reply.id} as='li'>
-                  <CommentBlock
-                    commentId={reply.id}
-                    parentCommentId={rootComment.id}
-                  />
-                </Flex>
-              ))}
-            </Flex>
-          )}
-
-          {hasMoreReplies && !hiddenReplies[rootComment.id] ? (
-            <PlainButton
-              onClick={handleLoadMoreReplies}
-              variant='subdued'
-              css={{ width: 'max-content' }}
-              disabled={isFetchingReplies}
-            >
-              {messages.showMoreReplies}
-              {isFetchingReplies ? (
-                <LoadingSpinner css={{ width: 20, height: 20 }} />
+          <Box
```

Comment:
interesting solution, where you get this from?

Reply:
I thought that flex box could do this, but after trying it didn't work out so after some googling i saw that grid worked in the way that I wanted